CHANGELOG.md

```diff
@@ -6,10 +6,23 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
```

Comment:
It seems like below ## Unrelaesed it should say "Nothing"?

See

https://github.com/cyberark/conjur/blob/271f85b3d472ace1e61a7bd115df711afab267fa/CHANGELOG.md

Reply:
Based on the example [here](http://keepachangelog.com/en/1.0.0/) it should be blank.